src/service/action-impl.js

```diff
@@ -49,6 +49,21 @@ const ELEMENTS_ACTIONS_MAP_ = {
   'AMP': ['setState'],
 };
 
+/** @const {!Object<string, !Object<string, string>>} */
+const WHITELISTED_INPUT_DATA_ = {
+  range: {
+    min: 'number',
+    max: 'number',
+    value: 'number',
```

Comment:
Nit: You can use an int-based enum to reduce the number of string constants in the min build.